generator/vulkan/render.zig

```diff
@@ -978,23 +961,25 @@ fn Renderer(comptime WriterType: type) type {
                 \\pub fn {s}Wrapper(comptime cmds: anytype) type {{
                 \\    comptime var fields: [cmds.len]std.builtin.TypeInfo.StructField = undefined;
                 \\    inline for (cmds) |cmd, i| {{
-                \\        const cmd_camel_case = snakeToCamel{s}(cmd);
-                \\        const cmd_type_name = "Pfn" ++ cmd_camel_case;
+                \\        const cmd_name = {s}CommandToString(cmd);
```

Comment:
I decided to wrap `std.meta.tagName` in a utility function here, as that ensures `anytype` is the actual, correct type. Just doing an `assert` produces a much less clear error.